### PR TITLE
More wrapper cleanup

### PR DESCRIPTION
**Pull request by drenalin23**
# Description
Remove the local copies of container.cfg and log4j2.xml templates.  Update some tests and prepare for a 0.5.5 tag.

### CODE REVIEW

File: recipes/filter-keystone-v2.rb [hunk 1]
```diff
@@ -27,3 +26,5 @@
 if %w(dev).include?(node.chef_environment)
   node.default['repose']['keystone_v2']['uri'] = 'http://localhost:8900/identity'
 end
+
+include_recipe 'repose::filter-keystone-v2'
```
Comment (drenalin23) [inline recipes/filter-keystone-v2.rb]: Why is this moved to the bottom?

Comment (drenalin23) [inline recipes/filter-keystone-v2.rb]: My concern being if you are worried about the order that the attributes are compiled in - I don't believe the order matters as that is all converged before the run.  If it did matter - we don't want the defaults from the upstream recipe coming *after* our logic to set them.  In an ordered world - we want defaults first, then set our own after that so we end up with the wrapper attributes at the end.

Comment (dimtruck) [inline recipes/filter-keystone-v2.rb]: @drenalin23 so the staging kitchen test that I wrote, when the include was at the top, failed because it showed the default template being laid down (you can replicate that with `kitchen verify staging-dfw1-keystone-ubuntu-1404`)

What you're saying makes complete sense but where do we overwrite the template again?  Maybe I'm not entirely certain how chef works as far as templates and attributes from wrapper cookbooks are concerned.  My change ended up fixing the template having default data problem but I'm hoping it's something I missed?

Comment (dimtruck) [inline recipes/filter-keystone-v2.rb]: @drenalin23 because I'm completely uncertain of what exactly is going on here, I'll revert the change to this file (I'll move the include back at the top) and you can take a look at the proper fix.  I'll keep the tests there for verification that we get proper attributes in keystone-v2 file.

File: .kitchen.yml [hunk 2]
```diff
@@ -33,3 +38,24 @@ suites:
       client_auth:
         auth_provider: 'OPENSTACK'
       rewrite_host_header: false
+- name: staging_dfw1_keystone
+  run_list: [
+    "apt",
+    "java",
+    "recipe[ele-repose::default]",
+    "recipe[ele-repose::filter-keystone-v2]",
```
Comment (shawnashlee) [inline .kitchen.yml]: does the trailing comma matter here?
